### ISSUE TITLE
Empty tag

### ISSUE THREAD

**Issue by ismamz** [posthtml/posthtml#89]
I want to add a empty alt tag.

I expect this:
```
alt=""
```

But, I wrote this:
```
node.attrs.alt = '';
```
or
```
node.attrs.alt = true;
```

And I get only this:
```
alt
```
This is the plugin: https://github.com/ismamz/posthtml-alt-always

**Comment by voischev** [comment]
Why you want this?
If open in browser
```html
<div alt="">test</div>
```
Browser parse this as
![screen shot 2015-12-17 at 23 57 15](https://cloud.githubusercontent.com/assets/1510217/11881788/0f093818-a51a-11e5-9170-fe1ac7b8f3af.png)

**Comment by ismamz** [comment]
I want this because for accessibility I need put a empty alt for images.
It's true that for browsers ```alt="" == alt``` and for many validators too.
For the purposes of my work, I find it more useful with doble quotes (```alt=""```).
Is there any alternative?

**Comment by voischev** [comment]
fix this in [posthtml-render@v1.0.5](https://github.com/posthtml/posthtml-render/commit/c47064b9f431c70141cd07297606e8b211abf236)

use new render module
